aif360/sklearn/datasets/seaphe_datasets.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Defines a class for the SEAPHE datasets.
+http://www.seaphe.org/databases.php
+"""
+import os
+import pandas as pd
+import numpy as np
+import tempfile
+import requests
+import zipfile
+from utils import standardize_dataset
+from sklearn.model_selection import train_test_split
+
+
+def load_lawschool_data(target, subset="all", usecols=None, dropcols=None,
```

Comment:
can we keep this named `fetch_lawschool_gpa` and have it replace the existing function in `aif360/sklearn/datasets/tempeh_datasets.py`?

Reply:
I have modified it's name to "fetch_lawschool_gpa" and have placed in "aif360/sklearn/datasets/tempeh_datasets.py" file. However, there's some issue in imports(which I have explained below) so, it's still referring to older "fetch_lawschool_gpa" function format and definition.

I tested multiple times in my local and "fetch_lawschool_gpa" always redircts to the updated method but when I run the notebook, it gives an error that "target is not a defined positional argument" which means it is referring to older "fetch_lawschool_gpa" method as older one didn't have "target" argument.

Reply:
I meant keep the tempeh_datasets.py file as is just rename the function in seaphe_datasets.py as `fetch_lawschool_gpa` then change the `__init__.py` to import from seaphe_datasets instead of tempeh_datasets. Once we know it's working, you can delete the tempeh_datasets.py file.

Did you test this after installing with the `--editable` flag?

Reply:
Oh Sorry! I have reverted changes in" tempeh_datasets.py" and renamed the method in "seaphe_datasets" and changed the __init__.py file to import the method from  "seaphe_datasets".

It is importing fine now, but I think the "seaphe_datasets" download issue is still not fixed. It is not returning zip file as expected in this call :-
requests.get("http://www.seaphe.org/databases/LSAC/LSAC_SAS.zip")

Reply:
I did "pip install --editable ." at root AIF360. And then launched jupyter-notebook from root. 

But these 2 imports :-
```
from aif360.sklearn.metrics import difference
from aif360.sklearn.inprocessing import GridSearchReduction
```
are still giving this error:_
`ModuleNotFoundError: No module named 'inFairness'`